pywt/multilevel.py

```diff
@@ -220,3 +220,156 @@ def waverec2(coeffs, wavelet, mode='sym'):
         a = idwt2((a, d), wavelet, mode)
 
     return a
+
+
+def iswt(coeffs, wavelet):
+    """
+    Multilevel 1D Inverse Discrete Stationary Wavelet Transform.
+
+    Parameters
+    ----------
+    coeffs : array_like
+        Coefficients list of tuples::
+
+            [(cA1, cD1), (cA2, cD2), ..., (cAn, cDn)]
+
+        where cA is approximation, cD is details, and n is start_level.
+    wavelet : Wavelet object or name string
+        Wavelet to use
+
+    Examples
+    --------
+    >>> import pywt
+    >>> coeffs = pywt.swt([1,2,3,4,5,6,7,8], 'db2', level=2)
+    >>> pywt.iswt(coeffs, 'db2')
+    array([ 1.,  2.,  3.,  4.,  5.,  6.,  7.,  8.])
+    """
+
+    output = coeffs[0][0].copy()  # Avoid modification of input data
```

Comment:
@grlee77 you may want to delete these comments, otherwise they stay visible with diff in the PR conversation